drivers/spi/spi_esp32_spim.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SPI_ESP32_SPIM_H_
+#define ZEPHYR_DRIVERS_SPI_ESP32_SPIM_H_
+
+#define MISO_IDX_0 HSPIQ_IN_IDX
+#define MISO_IDX_1 VSPIQ_IN_IDX
+#define MOSI_IDX_0 HSPID_OUT_IDX
+#define MOSI_IDX_1 VSPID_OUT_IDX
+#define SCLK_IDX_0 HSPICLK_OUT_IDX
+#define SCLK_IDX_1 VSPICLK_OUT_IDX
```

Comment:
I would suggest to move contents of this header to `spi_esp32_spim.c` file. There is no reason to keep it separate, as nobody else uses those definitions.